.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
I've met in some placed that option disabled for markdown files:

```
[*.md]
trim_trailing_whitespace = false
```

However, I'm not sure if it was a copy-paste from one source or there is a valid use case where new whitespace in a markdown files is required.